lin_sys/direct/pardiso/pardiso_interface.c

```diff
@@ -207,6 +203,12 @@ c_int init_linsys_solver_pardiso(pardiso_solver    **sp,
     s->iparm[34] = 0;     // Use Fortran-style indexing for indices
     /* s->iparm[34] = 1;     // Use C-style indexing for indices */
 
+#ifndef DFLOAT
+    s->iparm[27] = 0;
```

Comment:
Please add a comment describing what this parameter does (see above for other parameters).